### PR TITLE
validation-hint & form updates

### PR DESCRIPTION
**Pull request by Ddungyov**

### CODE REVIEW

File: data/tokens.json [hunk 1]
```diff
@@ -3000,16 +2972,16 @@
...
+              "spread": "4",
+              "color": "{Primary.gtmds_color_primary-focus}",
               "type": "innerShadow"
             },
             {
               "x": "0",
               "y": "0",
               "blur": "0",
-              "spread": "4",
-              "color": "{Primary.gtmds_color_primary-focus}",
```
Comment (StefanIvanov) [inline data/tokens.json:3012]: All above this line are changes that shouldn't be there ...

**Review by StefanIvanov — CHANGES_REQUESTED** [review]

File: data/tokens.json [hunk 2]
```diff
@@ -3093,10 +3089,6 @@
           "value": "{Success.gtmds_color_success-2}",
           "type": "color"
         },
-        "gtmds_hint_color": {
```
Comment (StefanIvanov) [inline data/tokens.json:3096]: Shows me that you removed this, why?

File: data/tokens.json [hunk 3]
```diff
@@ -3117,10 +3109,6 @@
           "value": "{Error.gtmds_color_error-3}",
           "type": "color"
         },
-        "gtmds_hint_color": {
```
Comment (StefanIvanov) [inline data/tokens.json:3120]: This one also...
Maybe we need a quick huddle as I cannot follow what you wanted to achieve :)

**Comment by Ddungyov** [comment]
> @Ddungyov were you able to prepare the final commit we talked about last week?

Yes, I have committed a new update based on the "next" branch.

**Comment by StefanIvanov** [comment]
> > @Ddungyov were you able to prepare the final commit we talked about last week?
> 
> Yes, I have committed a new update based on the "next" branch.

But I see no commits here

**Comment by Ddungyov** [comment]
> 
it should be this one
[52a809e](https://github.com/gtmhub/figmaTokens/pull/88/commits/52a809e913cdc71e5b1ca44de8f0e29afbcabce4)

File: data/tokens.json [hunk 4]
```diff
@@ -3117,10 +3109,6 @@
...
-          "value": "{Error.gtmds_color_error}",
-          "type": "color"
-        },
         "gtmds_hint-message_color": {
```
Comment (StefanIvanov) [inline data/tokens.json]:
```suggestion
        "gtmds_hint_color": {
```

**Review by StefanIvanov — APPROVED** [review]

**Pull request merged**